examples/build.gradle

```diff
@@ -0,0 +1,121 @@
+plugins {
+    id 'java'
+}
+
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation project(path: ":quartz")
+    implementation project(path: ":quartz-jobs")
+
+    implementation "org.slf4j:slf4j-api:$slf4jVersion"
+    implementation "org.slf4j:slf4j-log4j12:$slf4jVersion"
+}
+
+
+task runExample1(type: Exec) {
```

Comment:
There's a `JavaExec` task type for exactly this. Better syntax here would be something like:
```groovy
tasks.withType(JavaExec).configureEach() {
  classpath = sourcsets.main.runtimeClasspath
  group = '...'
}

tasks.register('runExample1', JavaExec) {
  description = '...'
  mainClass = 'org.quartz.examples.example1.SimpleExample'
}
```